vissl/hooks/byol_hooks.py

```diff
@@ -83,13 +83,13 @@ def _build_byol_target_network(self, task: tasks.ClassyTask) -> None:
         # Restore an hypothetical checkpoint, else random initialize 
         if task.loss.checkpoint is not None:
             task.loss.load_state_dict(task.loss.checkpoint)
-        # else:
-        #     logging.info("Copying model parameters")
-        #     for param_q, param_k in zip(
-        #         task.base_model.parameters(), task.loss.target_network.parameters()
-        #     ):
-        #         param_k.data.copy_(param_q.data)
-        #         param_k.requires_grad = False
+        else:
+            logging.info("Copying model parameters")
+            for param_q, param_k in zip(
+                task.base_model.parameters(), task.loss.target_network.parameters()
+            ):
+                param_k.data.copy_(param_q.data)
+                param_k.requires_grad = False
```

Comment:
This is actually may be a pretty big miss -- hopefully this improves.

We weren't copying the model weights, nor were we stopping the gradients.

Previously we were not stopping gradients from target network